src/plugins/sockets/inGameEvents.js

```diff
@@ -193,7 +193,8 @@ export async function handleInGameEvents(evData) {
       // Show OpponentRequestedStalemateDialog if opponent requested stalemate
       //   and game is not yet over
       gameStore.consideringOpponentStalemateRequest =
-        !evData.victory.gameOver && evData.requestedByPNum !== gameStore.myPNum;
```

Comment:
`requestedByPNum` was only used for the stalemate request action, but we're standardizing it to use `playedBy` everywhere so now we need to additionally check the `moveType`.